js/actions/bookmarkActions.js

```diff
@@ -4,15 +4,18 @@
 
 'use strict'
 
+const Immutable = require('immutable')
 const siteUtil = require('../state/siteUtil')
 const appActions = require('./appActions')
 const windowActions = require('./windowActions')
 const eventUtil = require('../lib/eventUtil')
+const appStoreRenderer = require('../stores/appStoreRenderer')
 const {SWITCH_TO_NEW_TABS} = require('../constants/settings')
 const getSetting = require('../settings').getSetting
 
 const bookmarkActions = {
-  openBookmarksInFolder: function (allBookmarkItems, folderDetail) {
+  openBookmarksInFolder: function (folderDetail) {
```

Comment:
so that is one of the temporary cases where we are firing appActions from windowActions, but the long-term fix is to just handle it in the browser process and the window updates based on state, not an action